completions/ssh

```diff
@@ -1,52 +1,56 @@
 # ssh(1) completion                                        -*- shell-script -*-
 
-_comp_cmd_ssh__queries()
+_comp_cmd_ssh__compgen_queries()
 {
-    local queries=$(_comp_cmd_ssh__query "$1" help)
-    [[ $queries ]] || queries="cipher cipher-auth mac kex key key-cert
-         key-plain key-sig protocol-version compression sig ciphers macs
-         kexalgorithms pubkeyacceptedkeytypes hostkeyalgorithms
-         hostbasedkeytypes hostbasedacceptedkeytypes"
-    _comp_compgen -ac "${cur,,}" -- -W "$queries help"
+    local -a ret
+    _comp_compgen -v ret -i ssh query "$1" help
+    [[ ${ret-} ]] || ret=(cipher cipher-auth mac kex key key-cert
+        key-plain key-sig protocol-version compression sig ciphers macs
+        kexalgorithms pubkeyacceptedkeytypes hostkeyalgorithms
+        hostbasedkeytypes hostbasedacceptedkeytypes)
+    _comp_compgen -ac "${cur,,}" -- -W '"${ret[@]}" help"'
```

Comment:
```suggestion
    local -a _ret=("${ret[@]}")
    _comp_unlocal ret
    _comp_compgen -ac "${cur,,}" -- -W '"${_ret[@]}" help"'
```

Note: One might think we could have stored the results in `_ret` from the beginning, but it doesn't work because `_comp_compgen -v _ret` is not accepted. We need to use a not-underscored array name to receive results from the internal call of `_comp_compgen`.

Also, I think no `-a` should be made default for consistency with other generators.

Reply:
Or, assuming #1028 `_comp_compgen -U var`,
```suggestion
    _comp_compgen -ac "${cur,,}" -U ret -- -W '"${ret[@]}" help"'
```

Reply:
Move to #1028